libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -60,6 +75,49 @@ namespace SohImGui {
             ImGui_ImplWin32_Init(impl.dx11.window);
             break;
         }
+        kokiri_col[0] = std::clamp((float) CVar_GetS32((char*)"gTunic_Kokiri_Red", 30)/255, 0.0f, 1.0f);
```

Comment:
Is there a cvar for float instead of s32?

Reply:
Dear ImGui uses float values for its color picker, but the tunic color uses int.